README.md

```diff
@@ -36,6 +36,27 @@ Run API token Rails generator and follow instructions:
 ```bash
 bundle exec rails generate webflow_sync:api_token_flow
 ```
+### Configuration options
+
+In `config/initializers/webflow_sync.rb` you can specify configuration options:
+
+1. `webflow_site_id`
```

Comment:
There's also `api_token` I believe? Is there any other config option?

Reply:
Thanks! Added `api_token`, so all options are listed now.

This is what we have in the configuration file:
```rb
    attr_accessor :skip_webflow_sync, :webflow_site_id, :sync_webflow_slug
    attr_reader :api_token
```